_config.yml

```diff
@@ -19,7 +19,8 @@ keywords:    "coala, review, analysis, static, bears"
 description: "coala provides a common command-line interface for linting and fixing all your code, regardless of the programming languages you use."
 source_link: "https://github.com/coala-analyzer/coala/"
 favicon:     "https://raw.githubusercontent.com/coala-analyzer/coala-artwork/master/artwork/coala_bear.png"
-
+logo:        "img/coala_logo.svg"
```

Comment:
@jayvdb I tried doing that, but the image is not loading due to type.
We need to change `htaccess` I guess and add `svg=image/svg+xml` .
For the time being, is this good to go or you have a better option?